MovieApp/MovieApp/MovieDBDownloader.h

```diff
@@ -1,24 +1,19 @@
-//
-//  MovieDBDownloader.h
-//  MovieApp
-//
-//  Created by user on 01/10/16.
-//  Copyright © 2016 internshipABH. All rights reserved.
-//
-
 #import <Foundation/Foundation.h>
 #import "ItemsArrayReceiver.h"
+#import <UIKit/UIKit.h>
 
 typedef enum Criteria
 {
-    MOST_POPULAR,
-    TOP_RATED,
-    LATEST
+    MOST_POPULAR=0,
+    LATEST=1,
+    TOP_RATED=2
+    
 }Criterion;
 
 @interface MovieDBDownloader : NSObject
```

Comment:
Probably should be called something like DataProviderService, since not only movies fill be fetched, nor just downloaded (there will be PUT, POST and DELETE requests too).